spyder/plugins/ipythonconsole.py

```diff
@@ -418,7 +419,14 @@ def setup_page(self):
                           _("Height:")+"  ", " "+_("inches"),
                           'pylab/inline/height', min_=1, max_=20, step=1,
                           tip=_("Default is 4"))
-        
+        bbox_inches_box = newcb(
+                _("Set bbox_inches to 'tight' when plotting inline"),
+                'pylab/inline/bbox_inches',
+                tip=_("Set bbox_inches to 'tight' when plotting inline."
+                      " This can cause discrepancies between the displayed"

```

Comment:
For the calltip, I suggest something like, "Sets bbox_inches to "tight" when plotting inline with matplotlib. [When enabled/disabled], can cause discrepancies between the displayed version and the image when written out with savefig."

Reply:
Like it. Would suggest "and that created using savefig." as a cleaner ending; I wasn't happy with mine either.

Reply:
Thanks for the feedback @CAM-Gerlach , I appreciate it.